package/Dockerfile-windows.agent

```diff
@@ -1,16 +1,16 @@
 ARG SERVERCORE_VERSION
-FROM mcr.microsoft.com/windows/servercore:${SERVERCORE_VERSION}
+FROM mcr.microsoft.com/powershell:lts-nanoserver-${SERVERCORE_VERSION} AS BUILD
```

Comment:
Can we change SERVERCORE_VERSION to DRONE_WINDOWS_VERSION or something else? I'm not sure if SERVERCORE is technically the right one anymore. I'll lean on the Windows team's expertise here, but thought I'd flag this in case we should change it.